### ISSUE TITLE
Unable to spell/navigate through image descriptions (alt text) using Microsoft Edge

### ISSUE THREAD

**Issue by xtstoll** [nvaccess/nvda#8990]
### Steps to reproduce:
1. Open Microsoft Edge on Windows 10
2. Go to https://www.bbc.com/news/world-us-canada-46390368
3. Press G to quickly navigate to an image with a description
4. Use the left and right arrows to spell out the contents of the description

### Actual behavior:
NVDA jumps to the next element on the page

### Expected behavior:
NVDA reads letter-by-letter (like it does in Chrome or Firefox)

### System configuration:
Factory settings

#### NVDA Installed/portable/running from source:
Installed

#### NVDA version:
2018.3.2

#### Windows version:
Windows 10

#### Name and version of other software in use when reproducing the issue:
Microsoft Edge 41

#### Other information:
The happen with EPUB files with image descriptions as well.

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA?
No

**Comment by josephsl** [comment]
Hi, looks like you wanted to say something, but couldn’t figure out how to fill out the form. Is that right? Thanks.

**Comment by xtstoll** [comment]
@josephsl Sorry! Check out the issue now :)

**Comment by Brian1Gaff** [comment]
The way I tend to handle this very annoying template is to select it all, 
and copy it to my notepad. I then edit it there and then paste the whole 
thing back. If I try to edit  and use the template in the form  chances are 
that sluggishness etc emnds up making me  have overwritten bits and left 
gaps and all sorts of problems.
 Thus I think many people will find thise way of working better.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by Adriani90** [comment]
This works as expected in Edge Chromium. I am closing this issue as works for me since Edge Chromium will be distributed to most users through Windows 10 20 H1 update.

**Comment by xtstoll** [comment]
I retested with NVDA 2020.1 in Windows 10.0.17763 with Microsoft Edge 44.17763.831.0. The alt tags are read in full when navigating with the G key, but when you try to spell the alt tag with the right arrow, **it doesn't spell the alt tag**. In Chrome 81.0.4044.138, this works fine.
